.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
     rev: 'v2.2.6'
     hooks:
       - id: codespell
-        args: ['-L', 'hart,noteable']
+        args: ['-L', 'hart,noteable', '--skip', "*.spec.ts"]
```

Comment:
Looks like `afterAll` makes the pre-commit check fail: https://results.pre-commit.ci/run/github/33653601/1715676598.F47qygP7RKuJagZupcWLmA

![image](https://github.com/jupyter/notebook/assets/591645/89b1e935-9342-4b3c-a04c-eb32c2e62b49)

Adding `afterAll` to the list of ignored words (via `-L`) does not seem to help, but skipping all `.spec.ts` files does. Might be worth investigating more in a separate issue or PR.